apps/browser/qml/pages/components/BrowserListItem.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.0
```

Comment:
This needs a copyright header, and the other files need them updating for 2021 too.